Platform/ArrowlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -801,6 +802,12 @@ UpdateFspConfig (
       FspsConfig->PcieGen4EqPh1UpTxPreset[2] = 0x7;
       FspsConfig->PcieGen4EqPh1UpTxPreset[4] = 0x7;
       FspsConfig->Usb4CmMode                 = 0x0;
+      FspsConfig->PcieRpLtrEnable[0] = 0x1;
+      FspsConfig->PcieRpLtrEnable[2] = 0x1;
+      FspsConfig->PcieRpLtrEnable[3] = 0x1;
+      FspsConfig->PcieRpLtrEnable[4] = 0x1;
+      FspsConfig->PcieRpLtrEnable[5] = 0x1;
+      StoreRpConfig (FspsConfig);
```

Comment:
I dont find this function in ARLS project as well as in common code. Please add this function into ARL if needed.

Reply:
fixed by removing "StoreRpConfig()' from this PR now, will submit another patch to add this function in.